CONTRIBUTING.rst

```diff
@@ -130,7 +130,7 @@ suite across the supported python versions.
 
   # install test requirements
   cd py4j-python
-  pip install -r requirements.txt
+  pip install -r requirements-test.txt
```

Comment:
I think it's better to keep `requirements.txt` alone for now so we don't have to document about `py4j-web/requirements-doc.txt`

Reply:
Ok, but this snippet doesn't work for now. I will remove it, since is not connected with other stuff in PR, but can be fixed somewhere.